presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorBucketFunction.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.raptor;
+
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.block.BlockAssertions.createIntsBlock;
+import static com.facebook.presto.block.BlockAssertions.createLongsBlock;
+import static com.facebook.presto.block.BlockAssertions.createStringsBlock;
+import static com.facebook.presto.raptor.RaptorBucketFunction.bigintHashFunction;
+import static com.facebook.presto.raptor.RaptorBucketFunction.intHashFunction;
+import static com.facebook.presto.raptor.RaptorBucketFunction.varcharHashFunction;
+import static org.testng.Assert.assertEquals;
+
+public class TestRaptorBucketFunction
+{
+    @Test
+    public void testBigintHash()
+    {
+        assertEquals(bigintHashFunction().hash(createLongsBlock(454345325), 0), -1124888032984160314L);
```

Comment:
We should test `RaptorBucketFunction` as a whole rather than just the hash functions